chart/values.yaml

```diff
@@ -68,6 +68,12 @@ remoteStorages:
     writeBearerToken: ""
     # readBearerToken is an optional bearer token to use when querying data from readURL.
     readBearerToken: ""
+    # writeHeaders is an optional list of headers in form `header:value`, attached to every write request.
+    # multiple headers must be delimited by '^^': 'header1:value1^^header2:value2'
+    writeHeaders: ""
```

Comment:
It is better commenting out these settings in the example config in order to show that they are optional